src/main/java/com/rackspace/ceres/app/services/DownsampleProcessor.java

```diff
@@ -70,7 +70,7 @@
   private final SeriesSetService seriesSetService;
   private final QueryService queryService;
   private final DataWriteService dataWriteService;
-  private List<ScheduledFuture<?>> scheduled;
+  public List<ScheduledFuture<?>> scheduled;
```

Comment:
Why is this made public?

Reply:
as it was initialised with setupSchedulers, so I was needed to check its value in its test class.

https://github.com/racker/ceres/blob/a9ab833dc7b9d9e6dac693529e19ffae31c755e6/src/test/java/com/rackspace/ceres/app/services/DownsampleProcessorTest.java#L110